mypy/checkexpr.py

```diff
@@ -311,12 +312,8 @@ def check_typeddict_call_with_kwargs(self, callee: TypedDictType,
                     msg=messages.INCOMPATIBLE_TYPES,
                     lvalue_name='TypedDict item "{}"'.format(item_name),
                     rvalue_name='expression')
-            items[item_name] = item_expected_type
 
-        mapping_value_type = join.join_type_list(list(items.values()))
-        fallback = self.chk.named_generic_type('typing.Mapping',
-                                               [self.chk.str_type(), mapping_value_type])
-        return TypedDictType(items, set(callee.required_keys), fallback)
+        return callee
```

Comment:
Yes. Also, the deleted code no longer serves its original purpose because of changes to `TypedDict` type inference.